src/islenska/basics.py

```diff
@@ -226,11 +230,13 @@ def mark_to_set(mark: Union[str, Iterable[str]]) -> Set[str]:
         at = re.sub(r"expl", r"það", at)
         # (we also allow Greynir-style person variants
         # ('p1','p2' & 'p3', but don't change 'op2'))
-        at = re.sub(r"[^o]?p([123])", r"\1p", at)
+        at = re.sub(r"(?<=[^o])?p([123])", r"\1p", at)
```

Comment:
(I changed this to be `(?<!o)` (negative lookbehind assertion, instead of positive lookbehind assertion for something else than o) after formatting the files with black)